src/components/widgets/DefaultWidget.tsx

```diff
@@ -16,11 +16,11 @@ type Props = {
 export const DefaultWidget = ({ Icon, count, onPress, text }: Props) => {
   return (
     <TouchableOpacity onPress={onPress}>
-      <WrapperVertical style={styles.container}>
+      <WrapperVertical style={styles().container}>
         <WrapperRow center>
-          <Icon style={styles.icon} />
+          <Icon style={styles(!!count?.toString()).icon} />
```

Comment:
why are you doing `toString` to pass the value?

i also think that no function for styles could be a better solution. if there is no count, you do not need to apply the icon style with padding right but always padding bottom, correct?

what about:

```
<WrapperVertical style={styles.container}>
        <WrapperRow center>
          <Icon style={!!count ? styles.iconWithCount : styles.iconWithoutCount} />
```

Reply:
This is a different solution. Both have the same end result. But in the future I will use what you say to make it more understandable. 👍

Reply:
If `count` is 0, this will always be false. So I had to convert it to string first. Otherwise, if count is 0, `paddingRight: 0` becomes

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/621/commits/095c971020462743627d19c5c2fd946ea322aa22

Reply:
the rendered result is the same but the processed code is not. making the `StyleSheet` functional has a few downsides. linting is affected, you have to call the styles in a functional way even if you do not have to pass anything and the `StyleSheet` const is created all the time you call the function. this should be only a work around for larger problems.
in our case in this PR we want to have styles for two cases. this is not a big problem we need to solve with functional styles. please read about `StyleSheet` in a function, maybe you can find out other reasons you can share.

Reply:
Yes you are right, the code has been updated. 👍